test/intl402/DurationFormat/prototype/format/invalid-arguments-throws.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DurationFormat.prototype.formatRange
+description: >
+  "format" basic tests when arguments are not  temporalDurationLike Objects should throw TypeError exception.
+info: |
+  Intl.DurationFormat.prototype.format(duration)
+  (...)
+  3. Let record be ? ToPartialDuration(duration).
+---*/
+
+const df = new Intl.DurationFormat();
+
+assert.throws(TypeError, () => { df.format(undefined) });
+assert.throws(TypeError, () => { df.format(-12) });
+assert.throws(TypeError, () => { df.format(-12n) });
+assert.throws(TypeError, () => { df.format(1) });
+assert.throws(TypeError, () => { df.format(2n) });
```

Comment:
Assuming https://github.com/tc39/proposal-intl-duration-format/pull/92 gets merged, these will be `RangeError`s because they will convert to a string which is not a valid ISO 8601 duration string.

I'd also suggest testing things like `{}`, `null`, `true`, and `"bad string"`.